styles.css

```diff
@@ -1,3 +1,3 @@
 .hidden {
-    visibility: hidden;
-}
\ No newline at end of file
+  display: none;
```

Comment:
I switched from `display: none` to `visibility: hidden` thinking that would fix this bug, but it didn't because the issue seems to be not using a mutation observer, so I'm changing this back because it's a better user experience.